matRad_calcQualityIndicators.m

```diff
@@ -104,13 +105,13 @@
                 % Conformity Index, fieldname contains reference dose
                 VTarget95 = sum(doseInVoi >= 0.95*referenceDose); % number of target voxels recieving dose >= 0.95 dPres
                 VTreated95 = sum(relevantDose(:) >= 0.95*referenceDose);  %number of all voxels recieving dose >= 0.95 dPres ("treated volume")
-                QI(runVoi).(strcat('CI',num2str(referenceDose))) = VTarget95^2/(numOfVoxels * VTreated95); 
+                QI(runVoi).('CI') = VTarget95^2/(numOfVoxels * VTreated95); 
```

Comment:
The reference dose had to be removed so you can display everything in the table?